### ISSUE TITLE
Android blank app load performance (no splash screen)

### ISSUE THREAD

**Issue by Giwayume** [godotengine/godot#63966]
### Godot version

3.4.5.stable

### System information

Pixel 3a XL Android 9

### Issue description

If I tap on a blank Godot app to load it, it takes a second or two before it opens. Most other apps open immediately and show the splash screen, however I never see the splash. It's doing something for a few seconds then jumping straight into the empty game. This is a problem because it gives the user a chance to second guess whether or not they mis-tapped on the icon.

Is there any way to modify the Android export to immediately load the splash screen, then start loading whatever is hanging the app up from loading immediately?

### Steps to reproduce

1. Create GLES2 Godot project
2. Modify `default_env.tres` to change background mode to `Clear Color` (remove sky generation)
3. Create a default empty scene
4. Project -> Install Android Build Template
5. Export using custom build (APK), as "Release" version.

**Comment by SkanerSoft** [comment]
I join the issue! It was also interesting when I noticed this on one of the test phones, but did not attach any importance.

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/30723.

**Comment by Giwayume** [comment]
Not so sure, this is about the export itself, since the Godot project is doing nothing (all engine code).

**Comment by SkanerSoft** [comment]
If I understand correctly, the problem is that when you start the game, instead of the splash screen, just a black screen is displayed.
At least that's how it was on one of my phones. Instead of my own picture, I saw darkness. But sometimes my picture flashed there, in rare cases. I figured it was a performance issue on the phone.

**Comment by Giwayume** [comment]
No, in my case nothing visibly occurs after tapping the icon, until about 2 seconds later the game opens to the empty scene. No splash screen is displayed.
